packages/deployment-service/src/functions/pipeline/pipeline-setup.ts

```diff
@@ -81,7 +79,7 @@ export const pipelineSetup: SQSHandler = async (event: SQSEvent, context: Contex
               Quantity: 1,
               Items: [`${pipeline.subDomain}.${process.env.ROOT_DOMAIN}`],
             },
-            Comment: `Cloudfront distribution for pipeline [${pipeline.id}]`,
+            Comment: `Cloudfront distribution for pipeline [${pipeline.id}] [${process.env.NODE_ENV === 'production' ? 'prod' : 'dev'}]`,
```

Comment:
Should be updated right?

Reply:
This is added to the lambda using env prop on lambda creation in cdk setup